conversations.yaml

```diff
@@ -374,6 +384,30 @@ components:
             notification_type:
               type: string
               enum: [REGULAR, CUSTOM]
+    EndOfConversationSignal:
+      properties:
+        recipient:
+          properties:
+            id:
+              type: string
+              description:
+                An identifier unique to the customer, such as phone number,
+                email, or another channel-specific value. For phone numbers,
+                the identifier must be in E.164 format.
+        sender_action:
+          type: string
+          description: The value is always 'conversation_ended'.
```

Comment:
Should this be an enum?

Reply:
I was thinking the same thing. It would be something like: 
```
enum: [ conversation_ended  ]
```
?

Reply:
Yes, LGTM!